tools/fastqc.wdl

```diff
@@ -12,28 +12,29 @@ task fastqc {
         String prefix = basename(bam, ".bam")
         Int memory_gb = 4
         Int max_retries = 1
+        String wait_var = ""
```

Comment:
Definitely don't like this. Can't think of a better option at the moment though.

Reply:
I implemented the wait on `picard validate` for the QC pipeline, but we don't validate the input bam in `rnaseq-standard`, so I left it using `wait_var`. It's only 2 tasks using it. It might make sense to validate the input bam before we start splitting it into fastqs. And if we did that, we could completely remove all `wait_var`s